src/manifest.ts

```diff
@@ -71,7 +74,7 @@ export async function manifest (args: string[]) {
     console.log(manifest)
   } else {
     const outFile = parsedArgs.out || outFilepath
-    Deno.writeTextFileSync(outFile, manifest)
+    Deno.writeTextFileSync(String(outFile), manifest)
```

Comment:
I'm not a big fan of this `String(outFile)` style casting because it requires doing it each time the variable is used.

Look at lines 58 and 59 for an example.

Is it possible to do this instead?

```js
const outFile: string = parsedArgs.out || outFilepath
```

Admittedly, I am only really familiar with Flowtype, not TypeScript, but whatever the TypeScript equivalent is.

Same question for lines 58 - 59 above for `slim`. Could something along these lines be done instead?

```js
const slim: string | undefined = parsedArgs.slim || parsedArgs.s
```

Reply:
parsedArgs values are typed as `unknown` by default, so direct assignment to `string` fails type checking; we use type assertions to tell TypeScript they're strings.
https://github.com/okTurtles/chel/pull/47/commits/ceae3abd58e0aa3baead94ed03c58b80f7c6cebb